parm/config/config.fcst

```diff
@@ -337,13 +340,13 @@ elif [[ "$CDUMP" == "gfs" ]] ; then # GFS cycle specific parameters
 fi
 
 if [ $DO_AERO = "YES" ]; then # temporary settings for aerosol coupling
-    export DIAG_TABLE="$HOMEgfs/parm/parm_fv3diag/diag_table_aer"
+    export DIAG_TABLE_APPEND="$HOMEgfs/parm/parm_fv3diag/diag_table_aer"
```

Comment:
diag_table_aer and diag_table_cpl are both full diag tables, not something to be appended.

Reply:
Understood